helm/trino-gateway/values.yaml

```diff
@@ -0,0 +1,126 @@
+replicaCount: 1
+
+image:
+  repository: "trinodb/trino-gateway"
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
+
+imagePullSecrets: []
+
+# Provide configuration for the Trino Gateway dataStore in dataStoreSecret. This node can
+# be left undefined if dataStore is defined under the config node. For production deployments
+# sensitive values should be stored in a Secret
+dataStoreSecret:
+    name: "config-yaml"
+    key: "config.yaml"
+
+# Provide configuration for the Trino Gateway backendState in backendStateSecret. This should
+# be used with health check configurations that require backend credentials. This node can
+# be left undefined if dataStore is defined under the config node.
+backendStateSecret:
+    name: ""
+    key: ""
+
+# Provide configuration for the Trino Gateway authentication configuration in authenticationSecret.
+# This node can be left undefined if dataStore is defined under the config node.
+authenticationSecret:
+    name: ""
+    key: ""
+
+config:
+    logging:
+        type: "external"
+    requestRouter:
+        #SQL clients connect to the request port
+        port: 9085
+        name: testTrinoRouter
+        historySize: 1000
+    server:
+        applicationConnectors:
+            - type: http
+              port: 9081
+              useForwardedHeaders: true
+        adminConnectors:
+            - type: http
+              port: 9082
+              useForwardedHeaders: true
+    clusterStatsConfiguration:
+        monitorType: INFO_API
+    modules:
+        - io.trino.gateway.ha.module.HaGatewayProviderModule
+        - io.trino.gateway.ha.module.ClusterStateListenerModule
+        - io.trino.gateway.ha.module.ClusterStatsMonitorModule
+    managedApps:
+        - io.trino.gateway.ha.GatewayManagedApp
+        - io.trino.gateway.ha.clustermonitor.ActiveClusterMonitor
+
+service:
+  type: ClusterIP
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: {}
+  # limits:
+  #   cpu: 1
+  #   memory: 1Gi
+  # requests:
+  #   cpu: 1
+  #   memory: 1Gi
```

Comment:
Do you suggest scale up or scale out in general?
For us, i notice that there is cpu throttle issue if cpu is set too low. so we prefer to scale up before scaling out. 
I haven't tested with recent version though

Reply:
I haven't experimented with running on less than 2 CPU. I run with at least 2 instances to avoid a SPOF, and usually use a c7g.xlarge or similar. 

We could set some reasonable limits by default, like 2 CPU/ 4Gi

Reply:
Sure sounds good

Reply:
Yeah .. higher, more realistic values for a production setup are probably better